kedro-datasets/RELEASE.md

```diff
@@ -1,7 +1,23 @@
 
-# Upcoming Release:
+# Upcoming Release 1.1.0:
+
+
+## Major features and improvements:
+
+* Added the following new datasets:
+
+| `polars.CSVDataSet` | A `CSVDataSet` backed by [polars](https://www.pola.rs/), a lighting fast dataframe package built entirely using Rust. | `kedro_datasets.polars` |
```

Comment:
I would keep the headers for the table. That's how we always format dataset additions in the Kedro release notes. 

```suggestion
| Type                                 | Description                                                                | Location                      |
| ------------------------------------ | -------------------------------------------------------------------------- | ----------------------------- |
| `polars.CSVDataSet` | A `CSVDataSet` backed by [polars](https://www.pola.rs/), a lighting fast dataframe package built entirely using Rust. | `kedro_datasets.polars` |
```

Reply:
Oh, I thought it was just an explanation of the structure. Mybad!